src/test/java/org/broadinstitute/hellbender/utils/pairhmm/VectorPairHMMUnitTest.java

```diff
@@ -23,22 +24,24 @@
 
     private static final String pairHMMTestData = publicTestDir + "pairhmm-testdata.txt";
 
-   @BeforeClass
-    public void initialize() {
-       try {
-           new VectorLoglessPairHMM(null);
-       } catch (final Exception e) {
-           throw new SkipException("AVX PairHMM is not supported on this system or the library is not available");
-       }
-    }
-
+    // Return a list of supported VectorLoglessPairHMM implementations, skip the test if none are supported
     private List<N2MemoryPairHMM> getHMMs() {
-        final PairHMMNativeArguments args = new PairHMMNativeArguments();
-        args.useDoublePrecision = false;
-        args.maxNumberOfThreads = 1;
-        final N2MemoryPairHMM avxPairHMM = new VectorLoglessPairHMM(args);
-        avxPairHMM.doNotUseTristateCorrection();
-        return Collections.singletonList(avxPairHMM);
+        List<N2MemoryPairHMM> list = new ArrayList<>();
```

Comment:
I think this should be restructured so that skipped configurations always produce a skip.   
Right now if you skip some of the implementations than you don't know it because it never produces those tests.  You only get a skip if you can't run any of them.